Paintroid/src/androidTest/java/org/catrobat/paintroid/test/espresso/tools/StampToolIntegrationTest.java

```diff
@@ -88,6 +89,7 @@ public void setUp() {
 		toolReference = activity.toolReference;
 	}
 
+	@Ignore("Causes crashes on jenkins")
```

Comment:
simply ignoring the testcase does not seem like proper fix to me. it should rather be considered rewriting the test. maybe it can be improved

Reply:
we decided to fix the tests once the tool redesign is done.